docs/Caimake.md

```diff
@@ -0,0 +1,75 @@
+# Caimake
+
+## About the Porject
+
+caimake helps you to build your project appropriately.
+
+It follows the [Makefile Specification](Specification.md).
+
+## Getting Started
+
+### Installation
+
+#### 1. Download from Github release
+
+Download directly from GIthub [release](directly)
+
+#### 2. Use `go get`
+
+```shell
+go get -u github.com/caicloud/build-infra/...
+```
+
+### Requirements
+
+-   `make` v4.2.1 or higher
+-   `sed` GNU sed, v4.4 or higher
+-   `git` v2.10.1 or higher
+-   `docker` for building and pushing container
+-   `golang` for compiling golang project, v1.8.0 or higher
+
+### Initialization
+
+Getting started from a git repo, use `caimake init <language>` to initialize it. 
+The following languages are supported now:
+- go
+
+`caimake init` will scan your project:
+-   treat project's name as docker images prefix
+
+for `golang`, it will:
```

Comment:
You are using `go` in source code, let's use `golang` in all places?

```go
	cmd := cli.NewCommand(&cobra.Command{
		Use:   "go",
		Short: "make for golang",
		Run:   runHelp,
	})
```

Reply:
I have already changed all visible `golang` to `go`.
`go` is shorter and more concise than `golang`

Reply:
Then don't use `golang` here (we can change along with readme rewrite)